src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-sync-primary-zone/rgw-sync-primary-zone.component.html

```diff
@@ -1,15 +1,27 @@
 <ul class="pb-5">
-  <li><i [ngClass]="[icons.large2x, icons.reweight]"
-         class="pt-2"></i></li>
-  <li class="badge badge-info mt-2">{{realm}}</li>
-  <li><i [ngClass]="[icons.large2x, icons.down]"
-         class="mt-2"></i></li>
-  <li><i [ngClass]="[icons.large2x, icons.cubes]"
-         class="mt-2"></i></li>
-  <p class="badge badge-info mt-2">{{zonegroup}}</p>
-  <li><i [ngClass]="[icons.large2x, icons.down]"
-         class="mt-2"></i></li>
-  <li><i [ngClass]="[icons.large2x, icons.deploy]"
-         class="mt-2"></i></li>
-  <li class="badge badge-info mt-2">{{zone}}</li>
+  <li>
+    <svg [cdsIcon]="icons.reweight"
+         [size]="icons.size20"
+         class="cds-icon--size-07"></svg>
+  </li>
+  <li class="mt-2"><cds-tag class="tag-info">{{realm}}</cds-tag></li>
+  <li>
+    <svg [cdsIcon]="icons.down"
+         class=" cds-icon--size-07"></svg>
+  </li>
+  <li>
+    <i [ngClass]="[icons.large2x, icons.cubes]"
+       class="mt-2"></i>
+  </li>
+  <p class="mt-2"> <cds-tag class="tag-info">{{zonegroup}}</cds-tag></p>
+  <li>
+    <svg [cdsIcon]="icons.down"
+         class="cds-icon--size-07"></svg>
+  </li>
+  <li>
+    <svg [cdsIcon]="icons.deploy"
+         [size]="icons.size20"
+         class="cds-icon--size-07 cds-icon"></svg>
+  </li>
+  <li class="mt-2"><cds-tag class="tag-info">{{zone}}</cds-tag></li>
```

Comment:
most of the changes in this file doesn't exist in your original PR. so if you try to bring those unrelated changes here, chances are something is going to break. Can you only make the changes that are really needed for tags? the icon changes will require a backport of a different PR.